src/System.Runtime.Extensions/tests/System/Environment.GetEnvironmentVariable.cs

```diff
@@ -135,7 +135,6 @@ public void CanGetAllVariablesIndividually()
         }
 
 #if netstandard17
-        [ActiveIssue(14417)]
```

Comment:
This should be fine to remove, since it appears I didn't need to disable it.

But please add an innerloop test as part of this that verifies consuming GetEnvironmentVariables typed as IDictionary and another typed as IEnumerable.